python/ray/dashboard/client/src/pages/dashboard/features/makeFeature.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { NodeInfoResponse } from "../../../api";
+
+type ArrayType<T> = T extends Array<infer U> ? U : never;
+type Node = ArrayType<NodeInfoResponse["clients"]>;
+type Worker = ArrayType<Node["workers"]>;
+
+export const makeFeature = ({
```

Comment:
Can you add a comment on what does `makeFeature` do?

Reply:
Since this was a bit confusing, I opted to simplify things by removing this function and instead using separate components for the node-level and worker-level features.